core/arch/arm/plat-ls/main.c

```diff
@@ -161,3 +164,41 @@ void main_secondary_init_gic(void)
 {
 	gic_cpu_init(&gic_data);
 }
+
+#ifdef CFG_HW_UNQ_KEY_REQUEST
+
+#ifndef ASM
```

Comment:
`#ifndef ASM` is not needed (always true)

Reply:
Accepted.